templates/cli/lib/commands/init.js.twig

```diff
@@ -54,9 +54,17 @@ const initProject = async () => {
 }
 
 const initFunction = async () => {
-  let answers = await inquirer.prompt(questionsInitFunction)
+  let answers = await inquirer.prompt(questionsInitFunction);
 
-  if (fs.existsSync(path.join(process.cwd(), 'functions', answers.name))) {
+  if (!fs.existsSync(path.join(process.cwd(), 'functions'))) {
+    fs.mkdirSync(path.join(process.cwd(), 'functions'), {
+      recursive: true
```

Comment:
I dont think recursive is necessary here, is it? You are already in `cwd()` current-working-directory.

Reply:
Better be safe than sorry. Looks good